Mu2eG4/fcl/prolog.fcl

```diff
@@ -26,8 +26,8 @@ mu2eg4DefaultPhysics: {
     setMuHadLateralDisplacement: true // improves accuracy of boundary
                                       // crossing for muons and common charged hadrons
 
-    modifyEMOption4: false // to set 115 MeV transition energy for the msc models when using _EMZ
-    modifyEMOption0: false // to set 115 MeV transition energy for the msc models with "opt0"
+    // mscTransitionEnergy = 115. // uncomment to change the Geant4 deafult
```

Comment:
The commented out example setting should use the colon, not the equal sign.

Reply:
done